portfolio/src/main/webapp/script.js

```diff
@@ -62,4 +62,29 @@ function getJSONData(){
       }
 
   });
-}
\ No newline at end of file
+}
+
+function getComment(){
+  fetch('/comment')  
+  .then(response => response.json()) 
+  .then((coments) => {
+    checkNumberOfComments(coments);
+    let commentContainer = document.getElementById("comments");
+
+    coments.forEach((comment) => {
+      commentContainer.appendChild(createListElement(comment));
+    });
+   
+  });
+}
+function checkNumberOfComments(array){
+  if(array.length > 5 ){
```

Comment:
Why 5? That seems fairly arbitrary.

Reply:
this was not a requirement for this project, but I know if I did not add it, too much data would be stored in the cache and such

Reply:
Fair. But you should still make this clear. A good option is to define a well named constant at the top of this file (perhaps with more details in the comment), and use that.